### ISSUE TITLE
Improve hold interaction on item page

### ISSUE THREAD

**Issue by jim** [chicago-tool-library/circulate#1205]
On the item page, we show a box with some text in it above the button that allows someone to put an item on hold:

![Image](https://github.com/chicago-tool-library/circulate/assets/3331/abf4d491-75b1-4ffc-a27b-c7bd5432a550)

The text is fairly complicated, though, and if someone is logged in, we should be able to tell them if they'll be first in line or not.

So, let's add a conditional in the view template when folks are logged in and display this if there aren't any other holds on the item:

> Placing a hold means you would like to be on the waiting list to use this item. No one else is waiting on this item, so once you place it on hold, you'll have a week to come to the library and pick it up.

If there are holds on the item, let's display this:

> Placing a hold means you would like to be on the waiting list to use this item. There are currently $NUMBER people waiting to use this item, and you will be $NUMBER + 1. When it is your turn to borrow the item, we'll send you an email and you'll have one week to come pick it up.

When someone has the item on hold, change the heading of the box to say "You have this item on hold". If the person is the next in line, the text should say:

> You have until $HOLD_EXPIRES_AT to pick up this item.

Otherwise, the text should say:

> You are on the waitlist for this item. There are X people ahead of you on the list.

Also, replace the "Place Hold" button with one that says "Cancel hold". When clicked, it should cancel the hold. Make this button not the primary button color.

Additionally:
* Make the "schedule an appointment" link the primary button color
* When the user is redirected to the item page after placing a hold, see if we can have the "you have this item on hold" be scrolled into view.
